src/components/footer/footer.ts

```diff
@@ -13,33 +13,33 @@ export class FooterComponent extends LitElement {
     const { currentYear, STARTING_YEAR } = this;
 
     return html`
-     <footer class='container as-footer'>
+     <footer class="container as-footer">
```

Comment:
3fffdefbbcfb0b0f8b574ac8ce97b6a8c4d9f258 is option 2 to hardcode the section if to apply it like this with footer changes from original work. @thescientist13
<img width="1008" alt="Screen Shot 2022-01-09 at 12 05 32 AM" src="https://user-images.githubusercontent.com/42796226/148670979-c5771eab-fd4a-48a4-8902-a0b0753e3f06.png">